bash/scripts/get_latest_tag.sh

```diff
@@ -11,5 +13,12 @@ get-latest-tag() {
   else
     exit 1
   fi
+
+  # If tag already released, bail out
+  if curl -sf "https://versions.deis.com/v3/versions/stable/deis-${component}/${tag}"; then
```

Comment:
@bacongobbler that's definitely a valid concern.  How about I remove the `s` (silent) flag and add the `v` (verbose) flag in its stead so we get the full output of why the curl command might have failed?